source/dub/commandline.d

```diff
@@ -2736,7 +2736,7 @@ private bool addDependency(Dub dub, ref PackageRecipe recipe, string depspec)
 {
 	Dependency dep;
 	const parts = splitPackageName(depspec);
-	const depname = parts.name;
+	const depname = PackageName(parts.name); // TODO: const
```

Comment:
`parts.name` should already be PackageName

Reply:
Resolved via 168a9831.